.github/workflows/run_ib.yml

```diff
@@ -66,7 +66,10 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Run the cleaning script
-        run: ./gcp.sh
+        run: |
+          echo "${GOOGLE_APPLICATION_CREDENTIALS}" > /tmp/gcp_creds
+          GOOGLE_APPLICATION_CREDENTIALS="/tmp/gcp_creds"
```

Comment:
If you want to use this variable in `gcp.sh` you have to `export` it.

Reply:
ops, thanks!